recipe/meta.yaml

```
@@ -1,40 +1,56 @@
 {% set name = "responses" %}
-{% set version = "0.13.3" %}
+{% set version = "0.25.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 18a5b88eb24143adbf2b4100f328a2f5bfa72fbdacf12d97d41f07c26c45553d
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 01ae6a02b4f34e39bffceb0fc6786b67a25eae919c6368d05eabc8d9576c2a66
 
 build:
   number: 0
-  noarch: python
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
+  script: '{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vvv'
 
 requirements:
   host:
     - python
     - pip
-    - pytest
+    - wheel
```

```suggestion
    - setuptools
    - wheel
```
See https://github.com/getsentry/responses/blob/82d1e53f0ceee444c4208e19c2bad69e3a3780c0/setup.py#L14